hibike/hibike_process.py

```diff
@@ -168,15 +168,16 @@ def helper():
                         ], 0.1)
                 elif hm.devices[hm.uid_to_device_id(uid)]["name"] == "YogiBear":
                     set_interval_sequence([
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0.5)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 1.0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", -0.5)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", -1.0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0)])
-
-                        ], 0.75)
+                        ##This is the test to see if the motor drives correctly
+                        #make_send_write(pipeToChild, uid, [("duty_cycle", 0)]),
+                        #make_send_write(pipeToChild, uid, [("duty_cycle", -1.0)]),
```

Comment:
This was another mess because of the somewhat brutish rebase. Changed back to the generic test on Master. These specific tests will need to be rewritten for when we test each YogiBear